toolbox/util/githubClient.go

```diff
@@ -152,26 +182,9 @@ func (g GithubClient) CloseFailedPullRequests(prTitlePrefix, repo, baseBranch st
 		if !strings.HasPrefix(*pr.Title, prTitlePrefix) {
 			continue
 		}
-		combinedStatus, _, err := g.client.Repositories.GetCombinedStatus(
```

Comment:
Can you only close PR that has been open for say 24 hours. such that if we need to run the command manually we don't accidently close PRs in flight.

Reply:
Done